### PR TITLE
Activation

### PR DESCRIPTION
**Pull request by mzylowski**
@andrixnet @harrieklomp 
Maybe also you need change in translation...

We had a problem with user who activated the account and probably he forgot about this fact. He tried to activate his account again for few times.
I made change in polish translation in 'activation_code_failed'. Now user is informed to check his activation status. Maybe he can just log in.

### CODE REVIEW

File: lib/languages/pl.php [hunk 1]
```diff
@@ -461,11 +461,11 @@
...
+    'register_msg6' => 'Kod aktywacyjny został wysłany na twój adres E-Mail.<br/>W celu zakończenia procesu rejestracji proszę się kierować instrukcją dostarczoną na twój adres E-Mail.<br/><b>Uwaga! Niektóre serwery pocztowe odrzucają list z kodem aktywacyjnym z serwera OC PL. Jeśli nie otrzymałeś kodu w ciągu najbliższych godzinach prosimy o kontakt na adres <span class="email">{contact_mail}</span> (podaj nazwę użytkownika jaką użyłeś podczas rejestracji)',
+    'register_msg7' => 'Użytkownik, który założy lub zrobi wpis do logu nie może być usunięty. Jest możliwa tylko deaktywacja konta. Będą wtedy usunięte wszystkie dane e-mail z profilu ale nazwa użytkownika zostanie zachowana. Jeśli chcesz deaktywować swoje konto skontaktuj się z nami za pomocą swojego konta e-mail, a deaktywujemy Twoje konto. W przypadkach niejasnych prosimy o kontakt <span class="email">{contact_mail}</span>. Dziękujemy.',
```
Comment (kojoty) [inline lib/languages/pl.php]: @mzylowski,
wydaje mi się, że podawanie adresu email w tłumaczeniu jest niepoprawne
tłumaczenia powinny być niezależne od ustawień danego noda, bo są wspólne dla wszystkich nodów - tam powinna być zmienna szablonu {contact_email} i to ona w templacie powinna być ustawiana na odpowiednia wartość z konfiguracji...

Comment (mzylowski) [inline lib/languages/pl.php]: Ale czy właśnie tak nie jest?
Skasowałem hardkodowany adres z tłumaczenia i zastąpiłem konfigurowalnym {contact_mail}.
Zasugerowałem się innym tłumaczeniem, które już było w tym pliku.
Chodzi Ci o to że, {contact_mail} powinien być ustawiany przez tpl_var w activation.php?

Comment (kojoty) [inline lib/languages/pl.php]: sorki, późno już i coś mi się chyba śniło - jest OK :)

File: lib/languages/pl.php [hunk 2]
```diff
@@ -8,7 +8,7 @@
     'Edit' => 'Edycja',
     'back' => 'Wróć',
     'cache_menu' => 'Menu skrzynki',
-    'activation_code_failed' => 'Kod aktywacyjny lub adres e-mail jest niepoprawny.<br/>Aktywacja Twojego konta przebiegła niepoprawnie. Proszę sprawdzić, czy nie popełniłeś błędu. Jeśli problem nadal występuje, proszę zgłosić go na adres <span class="mail>{contact_mail}</span> <b>koniecznie podaj nazwę użytkownika i adres e-mail pod jakim zarejstrowałeś się w OC PL</b>.',
+    'activation_code_failed' => 'Wystąpił błąd podczas aktywacji konta! Potencjalne przyczyny:<br /> - Kod aktywacyjny lub adres e-mail jest niepoprawny.<br/>- Twoje konto już jest aktywne (próbowałeś się zalogować?)<br />Jeśli po sprawdzeniu poprawności danych problem nadal występuje, proszę zgłosić go na adres <span class="mail">{contact_mail}</span><b> . Koniecznie podaj nazwę użytkownika i adres e-mail pod jakim zarejstrowałeś się w OC PL</b>.',
```
Comment (andrixnet) [inline lib/languages/pl.php]: "OC PL" at the end of this line should also be taken out of settings. It is a node dependent information, not translation dependent.

File: lib/languages/pl.php [hunk 3]
```diff
@@ -461,11 +461,11 @@
     'password_confirm' => 'Potwierdź hasło',
     'register_msg1' => 'W celu rejestracji na Opencaching PL musisz posiadać rzeczywisty adres E-mail na który zostanie wysłany kod aktywacyjny. Możesz tylko jednego użytkownika zarejestrować na jeden adres E-mail.',
     'register_msg2' => 'Pola zaznaczone * są obowiązkowe.',
-    'register_msg3' => 'Jeżeli masz jakieś pytania, proszę skontaktować się z nami, zanim zarejestrujesz się.',
+    'register_msg3' => 'Jeżeli masz jakieś pytania, proszę skontaktować się z nami (<span class="email">{contact_mail}</span>), zanim się zarejestrujesz.',
```
Comment (andrixnet) [inline lib/languages/pl.php]: Does this refer to "sign-up" or "sign-in" ?

File: lib/languages/pl.php [hunk 4]
```diff
@@ -461,11 +461,11 @@
...
-    'register_msg7' => 'Użytkownik, który założy lub zrobi wpis do logu nie może być usunięty. Jest możliwa tylko deaktywacja konta. Będą wtedy usunięte wszystkie dane e-mail z profilu ale nazwa użytkownika zostanie zachowana. Jeśli chcesz deaktywować swoje konto skontaktuj się z nami za pomocą swojego konta e-mail, a deaktywujemy Twoje konto. W przypadkach niejasnych prosimy o kontakt ocpl @ opencaching.pl Dziękujemy.',
+    'register_msg6' => 'Kod aktywacyjny został wysłany na twój adres E-Mail.<br/>W celu zakończenia procesu rejestracji proszę się kierować instrukcją dostarczoną na twój adres E-Mail.<br/><b>Uwaga! Niektóre serwery pocztowe odrzucają list z kodem aktywacyjnym z serwera OC PL. Jeśli nie otrzymałeś kodu w ciągu najbliższych godzinach prosimy o kontakt na adres <span class="email">{contact_mail}</span> (podaj nazwę użytkownika jaką użyłeś podczas rejestracji)',
```
Comment (andrixnet) [inline lib/languages/pl.php]: "OC PL" at the in this line should also be taken out of settings. It is a node dependent information, not translation dependent.

File: lib/languages/pl.php (hunk 1 above)
Comment (andrixnet) [inline lib/languages/pl.php]: I am uncertain of the exact meaning.  This is what I adapted from Google Translate:

> Users that posted at least one log entry or published at least one cache can not be deleted. <br />It is however possible to deactivate your account. All your personal data will be removed (including your email address), but your username will be kept. If you want to deactivate your account, please contact us at <span class = "email">{contact_mail}</span>. Thank you.

File: lib/languages/pl.php (hunk 4 above)
Comment (andrixnet) [inline lib/languages/pl.php]: In english I added a reference to check your spam folder: 

> The activation code has been sent to your e-mail address. <br/> In order to complete the registration process, please follow the instructions provided in that email message. <br/> <b> Warning! Some mail servers may reject or mark as spam the activation email from our system. Please also check your Spam folder as well. If you do not receive the code within the next few hours, please contact us at <span class = "email">{contact_mail}</span>. <b>Be sure to include the username and e-mail address you registered with.</b>

**Comment by andrixnet** [comment]
English and romanian posted here https://github.com/andrixnet/opencaching-pl/commit/495949fcd2f7908676cf89250a4fe7352c27d97d
Some clarifications are still needed.

File: lib/languages/pl.php (hunk 1 above)
Comment (deg-pl) [inline lib/languages/pl.php]: Ja p*dzielę... kto to pisał...
"Jeśli nie otrzymałeś kodu w ciągu najbliższych godzinach prosimy o kontakt..."

Comment (mzylowski) [inline lib/languages/pl.php]: Przerabiam cały mechanizm aktywacji na klikanie w link. Przy okazji poprawię wszystkie tłumaczenia.

**Comment by mzylowski** [comment]
Hi @andrixnet 
We will back to translation thema in next few days, after my patchset for activation and registration. ;]

**Pull request merged**